pkg/kubelet/cm/cpumanager/topology/topology.go

```diff
@@ -137,33 +138,67 @@ func (d CPUDetails) CPUsInCore(id int) cpuset.CPUSet {
 	return b.Result()
 }
 
-// Discover returns CPUTopology based on cadvisor node info
-func Discover(machineInfo *cadvisorapi.MachineInfo) (*CPUTopology, error) {
+// Discover returns CPUTopology based on lscpu output
+func Discover() (*CPUTopology, error) {
+	out, err := exec.Command("lscpu", "-p").Output()
```

Comment:
What if running in `containerized` mode? This is not working.

Reply:
correct